src/builder.rs

```diff
@@ -932,6 +923,38 @@ fn build_with_store_internal(
 		Arc::clone(&logger),
 	));
 
+	let persister = Arc::new(Persister::new(
+		Arc::clone(&kv_store),
+		Arc::clone(&logger),
+		10, // (?)
```

Comment:
I also don't know what value to recommend here, but pending when @tnull takes a look, you could define a constant to hold this value.

Reply:
FWIW I agree with you @tnull.

Our node ran on 100 for a long time and is 1000 now, but you would probably want 100 for most people. I dunno your design philosophy about what to make configurable, but it's probably a small number of power users who would want something other than 100.